src/main/java/Duke.java

```diff
@@ -1,10 +1,88 @@
+        while (!line.equals("bye")) {
+            //If user inputted list, return list of items
+            if (line.contains("list")) {
+                System.out.println("Here are the tasks in your list:");
+                for (int i = 0; i < noOfTask; i++) {
+                    int listNumber = i + 1;
+                    System.out.println(listNumber + "." + list[i]);
+                }
+            //If user inputted done, then mark the task as done
+            } else if (line.contains("done")) {
+                int divider = line.indexOf(" ");
+                int taskNumber = Integer.parseInt(line.substring(divider + 1));
+                list[taskNumber - 1].setDone();
+                System.out.println("Nice! I've marked this task as done:");
+                System.out.println("  " + list[taskNumber - 1]);
+            //If user inputted deadline, then add the task under deadline cat
+            } else if (line.contains("deadline")) {
+                int dividerAction = line.indexOf(" ");
+                int dividerDate = line.indexOf("/");
+                list[noOfTask] = new Deadline(line.substring((dividerAction + 1),
+                        (dividerDate - 1)), line.substring(dividerDate + 4));
+                noOfTask++;
+                System.out.println("Got it. I've added this task:");
+                System.out.println("  " + list[noOfTask - 1]);
+                if ((noOfTask - 1) == 0) {
+                    System.out.println("Now you have " + noOfTask + " task in the list.");
+                } else {
+                    System.out.println("Now you have " + noOfTask + " tasks in the list.");
+                }
+            //If user inputted to do, then add the task under to do cat
+            } else if (line.contains("todo")) {
+                int divider = line.indexOf(" ");
+                list[noOfTask] = new Todo(line.substring(divider + 1));
+                noOfTask++;
+                System.out.println("Got it. I've added this task:");
+                System.out.println("  " + list[noOfTask - 1]);
+                if ((noOfTask - 1) == 0) {
+                    System.out.println("Now you have " + noOfTask + " task in the list.");
+                } else {
+                    System.out.println("Now you have " + noOfTask + " tasks in the list.");
+                }
+            //If user inputted event, then add task under event cat
+            } else if (line.contains("event")) {
+                int dividerAction = line.indexOf(" ");
+                int dividerDate = line.indexOf("/");
+                list[noOfTask] = new Event(line.substring((dividerAction + 1),
+                        (dividerDate - 1)), line.substring(dividerDate + 4));
+                noOfTask ++;
+                System.out.println("Got it. I've added this task:");
+                System.out.println("  " + list[noOfTask - 1]);
+                if ((noOfTask - 1) == 0) {
```

Comment:
I think it is better for you to use the "<=" sign here because listing when there is 0 task will cause the else condition to execute. You may also consider commenting on what this if-else block does so that its purpose is made clearer.

Reply:
Ok! Thanks for the advice